modules/rules_cue/0.4.0/MODULE.bazel

```diff
@@ -0,0 +1,21 @@
+module(
+    name = "rules_cue",
+    version = "0.4.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "platforms", version = "0.0.6")
+bazel_dep(name = "rules_go", version = "0.39.0")
+
+cue = use_extension("//cue:extensions.bzl", "cue")
+use_repo(
+    cue,
+    "cue_tool_toolchains",
+)
+
+register_toolchains("@cue_tool_toolchains//:all")
+
+bazel_dep(name = "gazelle", version = "0.30.0")
+
+go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
```

Comment:
@seh I only now saw that you are declaring Go dependencies, but aren't bringing them into scope with `use_repo` - do you still need them?

Reply:
There are actions that involve running a Go program defined in this repository, and Bazel builds that Go program, so I know that I need the _go.mod_ file to be present, but it doesn't require any other modules, deliberately so. This works without the `use_repo` call. Does that mean that I can eliminate use of the `go_deps` module extension?

Reply:
You shouldn't need a `go.mod` file unless you want non-Bazel Go builds to work. You definitely should not need to use the extension if you don't `use_repo` any repos from it.

Reply:
I see. I did run the Go program in question many times outside of Bazel while I was writing it, so having the _go.mod_ file there made sense at the time. I can experiment with removing it, though, as replacing it would be easy enough in the future.

Would you accept that change in a subsequent release of this module? Should we hold up this release for that change?

Reply:
You can keep the `go.mod` but remove the extension usage.

Anyway, that's not a reason to block this release.

Reply:
See seh/rules_cue@d7d227c76a6f519b8e1f3fe8608ebe39e22c26f7 for removing the use of the `go_deps` module extension, to be included in a future release.

Thank you for catching that flaw.